### PR TITLE
Feature/proxy endpoint link preview

### PR DESCRIPTION
**Pull request by VitaliiDenysiuk**
implement proxy endpoint for link preview requests

### CODE REVIEW

**Review by DavidKorystov — APPROVED** [review]
Everything works perfectly

**Review by fominmaksim — APPROVED** [review]
lgtm

File: src/routes/index.route.ts [hunk 1]
```diff
@@ -14,4 +15,6 @@ router.post("/ping", (req: Request, res: Response) => {
 router.use("/auth", authRoutes);
 // router.use("/users", usersRoutes); ...
 
+router.use("/proxyLinkPreview", proxyLinkPreviewRouter);
```
Comment (alexandrtovmach) [inline src/routes/index.route.ts]:
```suggestion
router.use("/link-preview", proxyLinkPreviewRouter);
```

File: src/routes/proxyLinkPreview.route.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import express, { Request, Response } from "express";
+
+import validUrl from "../helpers/validUrl";
+import { getMetadata } from "../services/proxyLinkPreview.service";
+
+const proxyLinkPreviewRouter = express.Router();
+
+proxyLinkPreviewRouter.route("/").get(async (req: Request, res: Response) => {
```
Comment (alexandrtovmach) [inline src/routes/proxyLinkPreview.route.ts:8]:
```suggestion
proxyLinkPreviewRouter.route("/").get(async (req: Request<{ url: string }>, res: Response) => {
```

if I'm not wrong you can type query like this

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: src/routes/proxyLinkPreview.route.ts (hunk 2 above)
Comment (VitaliiDenysiuk) [reply src/routes/proxyLinkPreview.route.ts:8]: url in query.

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**